docs/content/en/docs/getting-started/configuration.md

```diff
@@ -75,3 +75,26 @@ directory into the `/etc/tetragon/tetragon.conf.d/` subdirectory. The latter is
 generally recommended. Defaults can be restored by simply deleting this file
 and all drop-ins.
 
+
+### Restrict gRPC API access
+
+The gRPC API supports unix sockets, it can be set using one of the following methods:
+
+1. Use the --server-address controlling setting directly:
+
+   `--server-address unix:///var/run/tetragon/tetragon.sock`
+
+2. Or use drop-in configuration file `/etc/tetragon/tetragon.conf.d/server-address` with a content of:
+
+   `unix:///var/run/tetragon/tetragon.sock`
+
+Note: Tetragon tarball ships with a default gRPC listening on `unix:///var/run/tetragon/tetragon.sock`.
+
+Then to access the gRPC API with `tetra` client, set `--server-address` to point to the corresponding address:
+
+   `sudo tetra --server-address unix:///var/run/tetragon/tetragon.sock getevents`
+
+Note: when reading events with `tetra` client, if its `--server-address` is not specified, it will
+try to guess if `Tetragon` daemon is running on the same host and use its `server-address` configuration.
```

Comment:
```suggestion
{{< note >}}
When reading events with the `tetra` client, if `--server-address` is not specified, it will 
try to detect if Tetragon daemon is running on the same host and use its `server-address` configuration.
{{< /note >}}
```